### PR TITLE
Correct expression typos on docs/aws.md

### PR DESCRIPTION
**Pull request by gpontesss**
Correct expression typos and enhance text on `docs/aws.md`.

Note: This is my first contribution to an open source project. I will appreciate suggestions and directions on what to do/not to do.

### CODE REVIEW

**Comment by gpontesss** [comment]
I did sign the CNCF CLA.

i think I'm supposed to it, also: 
/assign @justinsb

**Comment by gpontesss** [comment]
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.

Replying.

**Comment by mikesplain** [comment]
Thanks for this @gpontesss.  Before we can kick off tests we need to ensure the CLA is setup properly.

/check-cla

**Comment by gpontesss** [comment]
> Can you confirm you've followed these steps, the CLA is linked to this github account as well as the email address used to commit these commits?

I did as described. I also checked the commit email and the email registered on the CLA. I'll try again.

/check-cla

**Comment by justinsb** [comment]
Thanks and sorry for the CLA bot problems and then my delay in reviewing!

/approve
/lgtm
/ok-to-test

Would you mind rebasing - I can't resolve the conflict it looks like...

**Comment by rifelpet** [comment]
@gpontesss if you can rebase and fix the conflicts, we can get this merged soon

File: docs/getting_started/aws.md [hunk 1]
```diff
@@ -255,11 +255,11 @@ to revert or recover a previous state store.
...
+Information regarding cluster state store location must be set when using `kops` cli. See [state store](../state.md) for further information.
 
 ### Using S3 default bucket encryption
 
-kops supports [default bucket encryption](https://aws.amazon.com/de/blogs/aws/new-amazon-s3-encryption-security-features/) to encrypt the kops state in an S3 bucket. In this way, whatever default server side encryption is set for your bucket, it will be used for the kops state, too. You may want to use this AWS feature e.g. for easily encrypting every written object by default or when for compliance reasons you need to use specific encryption keys (KMS, CMK).
+`kops` supports [default bucket encryption](https://aws.amazon.com/de/blogs/aws/new-amazon-s3-encryption-security-features/) to encrypt its state in an S3 bucket. This way, the default server side encryption set for your bucket will be used for the kops state too. You may want to use this AWS feature , e.g., for easily encrypting every written object by default or when you need to use specific encryption keys (KMS, CMK), for compliance reasons.
```
Comment (johngmyers) [inline docs/getting_started/aws.md:262]:
```suggestion
`kops` supports [default bucket encryption](https://aws.amazon.com/de/blogs/aws/new-amazon-s3-encryption-security-features/) to encrypt its state in an S3 bucket. This way, the default server side encryption set for your bucket will be used for the kops state too. You may want to use this AWS feature, e.g., for easily encrypting every written object by default or when you need to use specific encryption keys (KMS, CMK) for compliance reasons.
```

**Pull request merged**